lib/flex/FlexController.js

```diff
@@ -58,9 +59,9 @@ class FlexController extends BaseController {
     this.projectSetup.setJobId(id);
     this.projectSetup.save((err) => {
       if (err) {
-        this.cliManager.log(LogLevel.ERROR, 'Failed to save job ID to project settings.');
+        this.cliManager.log(LogLevel.DEBUG, 'Failed to save job ID to project settings.');
```

Comment:
Because of a statement from the kinvey-cli-output doc saying that each command handler should output only debug messages while doing its job. When done, there should be either error or some data. Because a command should be either successful or not, IMO. If it achieves its main purpose, then it's successful.

Reply:
Discussed at length offline. Will be addressed in another PR.